docs/proposals/project-user-resource-quotas.md

```diff
@@ -26,32 +21,24 @@ Resources that will be under quota:
 
 ## Motivation and Background
 
-Clients would like to have options to fine-grain restrict resource usage per project, of a datacenter per project and individual resource usage.
+Clients would like to have options to fine-grain restrict resource usage per project.
 The reasons vary from being able to stop the depletion of resources in a shared datacenter, to being able to sell packages (projects) with a set 
 number of resources available. 
 
 This will be an EE feature.
 
 ## How will the resource quotas work
 
-Three types of resource quotas:
-- Project quota - amount of resources (vCPU, RAM, storage) that can allocated for nodes in a project
-- Project datacenter quota - amount of resources (vCPU, RAM, storage) that can allocated for nodes in a datacenter for a project. 
-- User quota - amount of resources (vCPU, RAM, storage) that can allocated for nodes when a user is creating them
-
+Project quota - amount of resources (vCPU, RAM, storage) that can allocated for nodes in a project.
 
 Rules:
 
 - The quotas are for node resources
+- The quotas are enforced(we should block creation of new nodes that would exceed the quota)
 - The resource quotas management will be admin-only.
-- By default, projects, project datacenters and users will not have a quota, meaning they can make as many resources they want
-- There will be a way to set a default resource quota for every project (should we do the same for users?)
-- When a user A creates a cluster in a project B, in datacenter C, the quotas for all of those are checked, and get filled up
-- For simplicity, the user which is the owner of a cluster, is used when calculating quotas for the cluster nodes. Even though another user can create nodes in that cluster.
-- When a node is deleted, the quota usage needs to decrease for:
-  - project
-  - the cluster owner
-  - project datacenter
+- By default, projects will not have a quota, meaning they can use as many resources they want
```

Comment:
Or the admin can configure a default quota. It is common practice with cloud providers as well (e.g. 100 vcpu, 400 GB RAM, 5000 GB Storage)

Reply:
The line just under it says this that admin can set a default quota :).